code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -42,12 +42,12 @@ paths:
     post:
       tags:
         - Phone number verify
-      summary: Verifies if the received hashed phone number matches the phone number associated with the access token
+      summary: Verifies if the received hashed/clear phone number matches the phone number associated with the access token
```

Comment:
**plain text** looks better than *clear*

Reply:
Edited here and couple of more pleaces.